warp/src/main/java/com/schibsted/nmp/warp/components/WarpButton.kt

```diff
@@ -172,13 +183,18 @@ fun WarpButton(
         val elevation =
             if (!loading && style == WarpButtonStyle.UtilityOverlay) dimensions.shadowSmall else 0.dp
 
-        val buttonModifier = if (loading) modifier.loadingAnimation() else modifier
         val colors = if (loading) loadingColors else buttonColors
         //disable click when button is in loading state
         val clickAction = if (loading) {
             {}
         } else onClick
 
+        val paddingVertical = if (small) dimensions.space025 else dimensions.space2
+        val paddingHorizontal = if (small) dimensions.space025 else dimensions.space2
+        val loadingModifier = if (loading) modifier.loadingAnimation() else modifier
+        val smallModifier =
```

Comment:
you could go with the `.then` conditional attritbute for Modifiers: https://medium.com/@sujathamudadla1213/module-4-custom-modifiers-advanced-topics-topic-modifier-then-174414608bcd

Reply:
I have used that :) Line 197